packages/nouns-api/src/utils.ts

```diff
@@ -43,3 +44,12 @@ export const getTokenMetadata = async (tokenId: string): Promise<undefined | Tok
 
   return metadata;
 };
+
+export const hasNounToken = async (account: string) => {
+  const data = await tryF(() => nounsTokenContract.getCurrentVotes(account));
+  if (isError(data)) {
+    console.error(`Error fetching votes for account ${account}: ${data.message}`);
+    return;
+  }
+  return data.votes?.toNumber();
```

Comment:
Is the idea to use `getCurrentVotes` as a bit of a proxy for if the user has a lil noun? Not totally sure how `getCurrentVotes` works, what if the user has never voted?

Reply:
Yeah i believe this is actually checking how many lil nouns they have and therefore how many votes they'd have. The idea here being we want to restrict user submissions to people with at least 1 lil noun so this should give us that data. Although I can't get the call to work locally.